base/gcutils.jl

```diff
@@ -38,18 +38,30 @@ Immediately run finalizers registered for object `x`.
 finalize(@nospecialize(o)) = ccall(:jl_finalize_th, Cvoid, (Ptr{Cvoid}, Any,),
                                    Core.getptls(), o)
 
+module GCUtils
+
+export gc, gc_enable
+
 """
     gc()
 
-Perform garbage collection. This should not generally be used.
+Perform garbage collection.
+
+!!! warning
+    This should not generally be used.
```

Comment:
I think this is too strong. We probably should have just said "Excessive use will likely lead to poor performance". Being "not used often" (appropriate wording) and "should not generally be used" (current wording) don't quite express the same level of concern.